ITE/8/README.adoc

```diff
@@ -0,0 +1,232 @@
+The purpose of Policy as Code is to codify policy definitions in software, which allows for consistent, automated assessment of policy compliance in modern software development practices.
+
+Increased security with reusable patterns and code snippets for creating additional policies.
+in-toto-library will allow the projects to move fast yet still keep information secure. By doing so, we can trust this library.
+
+=== Use Case 2: Providing Community-Owned Library of Policies
+
+link:https://github.com/open-policy-agent/gatekeeper-library[gatekeeper-library], would be a good benchmark for this use-case.
+Creating public library would allow us to manage policies easier and evaluating policies against different attestations using policy engines. We do not
+have to download policies.
+
+We can create a _source-of-truth_ by structuring information models and associated data schema such that every data element is mastered in only one place.
+
+=== Use Case 3: Verifying Attestations (OCI Registry)
+
+link:https://github.com/sigstore/cosign[cosign], is a container signing and verification tool, owned by link:https://github.com/sigstore/[sigstore]. Cosign also has built-in support for in-toto attestations.
+From the link:https://github.com/sigstore/cosign/issues/259[Support in-toto attestations] discussion, cosign got a new link:https://github.com/sigstore/cosign/pull/458[attest command].
+We able to create and sign local predicate files, and, also link:https://github.com/sigstore/cosign/blob/main/cmd/cosign/cli/verify_attestation.go[have ability] to verify them! It link:https://github.com/sigstore/cosign/pull/504/files[supports to generate]
+well-known attestation specs (i.e., SPDX, Provenance, Link). Our next steps would be [determinate attestation formats for link:https://github.com/sigstore/cosign/issues/442[vuln scans].
+We have been link:https://github.com/in-toto/attestation/issues/58[tracking] the aforementioned issue in the attestation repository.
+
+Verifying attestation formats using policies is quite link:https://github.com/sigstore/cosign/issues/512[new idea]. We link:https://github.com/sigstore/cosign/pull/641/[implemented] the attestation validation against CUE policies.
+This is the demo where this idea is coming from, and why this proposal should have created!
+
+> This is really new and exciting for all of the in-toto space!link:https://github.com/sigstore/cosign/pull/641#issuecomment-916970111/[*] -Dan Lorenc
+
+After short demonstration of how in-toto fits into cosign, we can extend the usability of in-toto attestation for different projects and tools.
+This is why in-toto will have critical role for the supply chain security world!
+
+
+[[reasoning]]
+== Reasoning
+
+Policy as Code eliminates the need for manual implementation with regard to sustaining and maintaining projects in in-toto.
```

Comment:
I think something to add here is that currently writing layouts (essentially policies for the current spec) is a semi-manual task at the moment, and this is feedback we've received from some users.